invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/side_bar/external_resources.html

```diff
@@ -0,0 +1,59 @@
+{#
+Copyright (C) 2020 CERN.
+
+Invenio RDM Records is free software; you can redistribute it and/or modify
+it under the terms of the MIT License; see LICENSE file for more details.
+#}
+{% set external_resources = record["ui"].get("related_identifiers", None) %}
+
+{%- if external_resources -%}
+  <section id="external-resource" aria-label="{{ _('External resources') }}" class="ui segment rdm-sidebar external resource">
+  <h2 class="ui small header">{{ _('External resources')}}</h2>
+  {# dynamically create the list of external resources #}
+  <ul class="details-list m-0 p-0" style="list-style-type: none;">
+    {%- for itr in config.get("APP_RDM_RECORD_LANDING_PAGE_EXTERNAL_LINKS") -%}
+      {%- for resource_itr in external_resources if resource_itr['scheme'] == 'url' and itr['id'] in resource_itr.get('identifier', []) -%}
+
+        {% set resource = itr.render(resource_itr) %}
+
+        {%- if resource -%}
+          {%- if resource.template -%}
+            {# It's up to the template to render the `resource` as it wishes #}
+            {%- include resource.template %}
+          {%- else -%}
+            {# Default rendering #}
+            <li class="has-popup">
+            <div class="resource clickable" tabindex="0" aria-haspopup="dialog"
+              aria-expanded="false" role="button" aria-label="{{ resource.content.title }}"
+              style="display:flex; flex-direction: row;">
+
+              {% if resource.content.icon %}
+                {% set iconFile = 'images/{}'.format(resource.content.icon) %}
+
+                <span class="icon-wrap">
+                  <img class="icon" src="{{ url_for('static', filename=iconFile) }}" alt="{{ resource.content.title }} icon" height="32" width="32" />
+                </span>
+              {% endif %}
+
+              <div style="display: flex; flex-direction: column; justify-content: space-around;">
```

Comment:
please don't set inline styling, these should live under the theme files. In some cases inline styling will be triggering content security policy errors.
There are already some classes defined to cover this styling. Check https://github.com/inveniosoftware/invenio-theme/tree/master/invenio_theme/assets/semantic-ui/less/invenio_theme/theme (global theme) 
and https://github.com/inveniosoftware/invenio-app-rdm/tree/master/invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme

Reply:
Ok I'll fix it!
Thx for the review also! :)